src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -137,9 +134,12 @@ export function CreateExportModal({
               )}
             </Stack>
             <Stack align="center" mt="xs" w="40%">
-              <Text size="md" fw={450}>
-                Creation date range
-              </Text>
+              <Group>
+                <Text size="md" fw={450}>
+                  Creation date range
+                </Text>
+                <InfoButton text="Only issues within the selected date range are exported" />
+              </Group>
```

Comment:
Again a reason to rebase onto #128.

The text could then read something like `Only issues whose in-progress time is completely inside this date range are exported.`